clients/fides-js/src/fides.ts

```diff
@@ -243,12 +263,12 @@ const init = async ({
         `User location could not be obtained. Skipping overlay initialization.`
       );
       shouldInitOverlay = false;
-    } else if (!effectiveExperience) {
+    } else if (!effectiveExperience || hasNoticesThatRequireConsent) {
```

Comment:
Really specific question here... are you sure this is required?

In a scenario where you detect `hasNoticesThatRequireConsent == true`, doesn't that require already having the `experience`? Are you sure it makes sense to refetch?

Reply:
This was accounting for a possible scenario where the user preference cookie was deleted or the fides BE is out of sync with the cookie for some reason. So we trust the BE by making a fetch for user preference. I'm OK just saying we trust the cookie 100% if you're OK with it?

Reply:
went ahead and removed it for simplicity